cmd/selfupdate.go

```diff
@@ -0,0 +1,66 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of K9s
+
+package cmd
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os"
+	"runtime"
+
+	"github.com/creativeprojects/go-selfupdate"
+	"github.com/rs/zerolog/log"
+	"github.com/spf13/cobra"
+)
+
+func selfUpdate() *cobra.Command {
+	command := cobra.Command{
+		Use:   "selfupdate",
+		Short: "Update k9s binary",
+		Long:  "Update k9s binary",
+		Run: func(cmd *cobra.Command, args []string) {
+			checkAndUpdateApp()
+		},
+	}
+
+	return &command
+}
+
+func checkAndUpdateApp() error {
+	latest, err := checkVersion()
+	if err != nil {
+		return err
+	}
+
+	if latest.LessOrEqual(version) {
+		log.Info().Msgf("Current version (%s) is the latest", version)
```

Comment:
Likely users of this command won't be looking at logs. Best to output the results to the console stdout or stderr.
Also thinking it might be good idea to match other k9s command look and feel ie banner + colorized output. 

BONUS: leverage a progress bar (https://github.com/schollz/progressbar) or other to provide feedback

Reply:
Great catch! Yes, I'll change it to use stdout instead of logs and try to add progressbar :)